w10h01/test/pgdp/trials/TrialOfTheDreamsTest.java

```diff
@@ -0,0 +1,59 @@
+package pgdp.trials;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static pgdp.trials.TrialOfTheDreams.lockPick;
+
+public class TrialOfTheDreamsTest {
+    // Please note: Method lockPick(Function<Byte[], Boolean> lock, List<Byte> key, int maxlen) has to be set to public
```

Comment:
[Please do not do this. Use reflection instead.](https://github.com/MaximilianAnzinger/pgdp2223-tests/blob/main/CONTRIBUTING.md#dont-depend-on-changing-the-code-of-the-implementation)